server/libs/infra/src/db/migrations/1679751316282-UpdateTranscodeOption.ts

```diff
@@ -0,0 +1,27 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class UpdateTranscodeOption1679751316282 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
```

Comment:
Should we delete the old key, or how would you handle the case it was set to false?

Reply:
I noticed that immich doesn't store the default value of `transcodeAll` (equals to `false`) into database. So `system_config` table could only store `transcodeAll` with `true` value

Reply:
Actually that does sounds correct, I forgot 😁. Perfect!